packages/wonder-blocks-color/index.js

```diff
@@ -18,4 +20,6 @@ const Color = {
     salmon: "#FD6C6E",
 };
 
+Color.azul = Color.blue;
```

Comment:
I think this is a mistake - it should just be "blue" (azul is blue in spanish, I don't think we necessarily need this!)